src/lib/JSONEditor.js

```diff
@@ -136,7 +136,7 @@ export default class JSONEditor extends React.Component {
             value={label}
             addElement={this.addElement}
             removeElement={this.removeElement}
-            showRemoveButton={this.props.showRemoveButton}
+          //showRemoveButton={this.props.showRemoveButton}
```

Comment:
Can we pass value instead!

Reply:
@Nikmahajan started to work on passing values as per our discussion.

Reply:
@Nikmahajan, Now no code is commented we have added more props to handle our case it is now controllable and values are being passed by props only.

![props are added for our cases](https://github.com/pepcus-Aish/SimpleJSONEditor/assets/141809285/8c070c2d-05ed-4447-864e-73218a514445)

we could see showAddButtonObject and ShowRemoveButtonObject is added. to handle lib as per any case.

![image](https://github.com/pepcus-Aish/SimpleJSONEditor/assets/141809285/8510ba56-5e29-4013-8532-5beb764f3a52)

The above code shows no code commented.